lib/enkf/tests/value_export.cpp

```diff
@@ -19,16 +19,65 @@
 #include <stdbool.h>
 #include <stdio.h>
 
+#include <algorithm>
 #include <iostream>
 #include <fstream>
+#include <string>
+#include <vector>
 
 #include <ext/json/cJSON.h>
 
+#include <ert/util/util.h>
 #include <ert/util/test_util.h>
 #include <ert/util/test_work_area.hpp>
 #include <ert/enkf/value_export.hpp>
 
 
+namespace {
+
+// This is super ugly but I could not find any way to do this without using a
+// a global variable. Of course the correct thing to do would be to implement
+// a new walk_directory that either collects first and return (include
+// max_depth, remove callbacks), or behaves like an iterator. But that would
+// require quite some efforts, and it's probably not worth it for just one
+// test. Especially considering that C++17 includes the filesystem library that
```

Comment:
Considering that `libres` is now moving towards having a Python-only API; I would say that we can use whatever compiler floats our boat ;)